src/libraries/AccountAccessParser.sol

```diff
@@ -135,8 +135,9 @@ library AccountAccessParser {
 
     bytes32 internal constant LIVENESS_GUARD_LAST_LIVE_SLOT = bytes32(uint256(0));
 
-
     bytes32 internal constant ANCHOR_STATE_REGISTRY_RETIREMENT_TIMESTAMP_SLOT = bytes32(uint256(6));
+    bytes32 internal constant ANCHOR_STATE_REGISTRY_PROPOSAL_ROOT_SLOT = bytes32(uint256(3));
+    bytes32 internal constant ANCHOR_STATE_REGISTRY_PROPOSAL_L2_SEQUENCE_NUMBER_SLOT = bytes32(uint256(4));
```

Comment:
I don't see anything at [slot 4 in the ASR](https://github.com/ethereum-optimism/optimism/blob/5701d5d956ce59079ed9904e4fb475f3786be59c/packages/contracts-bedrock/snapshots/storageLayout/AnchorStateRegistry.json#L43)?

Reply:
oh, I see [Proposal](https://github.com/ethereum-optimism/optimism/blob/5701d5d956ce59079ed9904e4fb475f3786be59c/packages/contracts-bedrock/src/dispute/lib/Types.sol#L44-L47) is 2 words long.